### ISSUE TITLE
ReferenceError: window is not defined

### ISSUE THREAD

**Issue by mushonnip** [blowstack/ckeditor-nuxt#9]
```
 ERROR  [Vue warn]: Failed to resolve async component: () => Promise.resolve(/*! import() */).then(__webpack_require__.t.bind(null, /*! @blowstack/ckeditor-nuxt */ "@blowstack/ckeditor-nuxt", 7))
Reason: ReferenceError: window is not defined
```

**Comment by blowstack** [comment]
where it's displayed? In browser console or in system console?

**Comment by mushonnip** [comment]
> where it's displayed? In browser console or in system console?

in the system console

**Comment by blowstack** [comment]
I assume it's shown when you refresh the page.
It shouldn't affect anything really. Does the editor work?

**Comment by mushonnip** [comment]
> I assume it's shown when you refresh the page.
> It shouldn't affect anything really. Does the editor work?

The editor is work but it's like rendered slowly

**Comment by blowstack** [comment]
Yes, it's quite slow because It has mplemented all free plugins from ckeditor (except ckfinder). You can try to exclude plugins that you don't need passing a proper config. For example to disable Title plugin you can pass sth like this:
   
 ``` javascript
 editorConfig: {
        removePlugins: ['Title'],
      }
```

If it doesn't help I recommend [making your own custom build with selected plugins](https://blowstack.com/blog/frameworks/create-ckeditor-5-custom-build) that way you shoud get a lighter version.

**Comment by mushonnip** [comment]
> Yes, it's quite slow because It has mplemented all free plugins from ckeditor (except ckfinder). You can try to exclude plugins that you don't need passing a proper config. For example to disable Title plugin you can pass sth like this:
> 
> ```js
> editorConfig: {
>        removePlugins: ['Title'],
>      }
> ```
> 
> If it doesn't help I recommend [making your own custom build with selected plugins](https://blowstack.com/blog/frameworks/create-ckeditor-5-custom-build) that way you shoud get a lighter version.

Can we get the state when the ckeditor is ready? For content loader usage

**Comment by andymark-by** [comment]
Have this problem too!
That what I see in terminal when use `npm run dev`?
![Снимок экрана 2021-02-02 в 10 48 15](https://user-images.githubusercontent.com/39381353/106582429-07a7e000-6555-11eb-9c1f-f13ab2b53da5.png)

**Comment by andymark-by** [comment]
> Have this problem too!
> That what I see in terminal when use `npm run dev`?
> ![Снимок экрана 2021-02-02 в 10 48 15](https://user-images.githubusercontent.com/39381353/106582429-07a7e000-6555-11eb-9c1f-f13ab2b53da5.png)

I find answer! Need added `process.client` check like this and error missing
`export default {
  components: {
    'ckeditor-nuxt': () => { if (process.client) { return import('@blowstack/ckeditor-nuxt') } },
  },
}`

**Comment by blowstack** [comment]
@andymark-by Good job. I've just merged your pull request. thx

**Comment by andymark-by** [comment]
@blowstack Thank you for the good module.
